SuperBuild.cmake

```diff
@@ -308,7 +304,7 @@ set(BRAINSTools_options
 
 Slicer_Remote_Add(BRAINSTools
   GIT_REPOSITORY ${EP_GIT_PROTOCOL}://github.com/Slicer/BRAINSTools.git
-  GIT_TAG d87ea8f357622c27abc6a2162b4623abe782b995 # slicer-2020-01-08-d0e6c01
+  GIT_TAG v5.2.0 # 2020-04-14
```

Comment:
Since no branch has been pushed to https://github.com/Slicer/BRAINSTools, `GIT_REPOSITORY` should be changed to BRAINSia/BRAINSTools

Reply:
Thanks, I see there are two tags. Which one is correct ? 

![image](https://user-images.githubusercontent.com/219043/79377739-27b8d500-7f2a-11ea-9b23-6902e5137ef5.png)

Reply:
I am assuming we can remove `5.2.0` ?